### PR TITLE
port scrabble-score + two-fer

### PR DESCRIPTION
**Pull request by jitwit**

### CODE REVIEW

**Comment by jitwit** [comment]
I guess I should add an `unlocked-by` or put it in core. Unlocked by `hello-world`?

**Comment by guygastineau** [comment]
Looks like it is unlocked by hamming.

That is good enough for me, although there might be a better fit.

**Comment by jitwit** [comment]
Hamming is like taking a list and summing up something about it, this is similar but slightly different logic

**Comment by guygastineau** [comment]
Yeah, I was wondering if there was another core exercise doing matching.

**Comment by guygastineau** [comment]
While we are never going to find a core exercise that teaches the same things as Scrabble score (in my opinion - they would have to be the same exercise) perhaps `rna-transcription` is a better fit.

**Comment by guygastineau** [comment]
I think that `hello-world` is good for unlocking `two-fer`

**Comment by jitwit** [comment]
I don't think it should matter too much, the choice is pretty arbitrary. As long as we have problems that unlock other problems that's all that matters

**Comment by guygastineau** [comment]
Well, the project in collaboration with the university of Chicago is going to be all about the anatomy of tracks, so those aspects will matter a lot to the analysis they will be doing.

I don't know when that starts exactly.  I think that either `hamming` or `rna-transcription` will be fine for `scrabble-score`.

File: code/config.ss [hunk 1]
```diff
@@ -131,16 +131,16 @@
      ((slug . two-fer)
       (uuid . "3ecc2d1c-55e0-45c9-ba35-57d7d8cfd51e")
       (core . #f)
-      (unlocked-by)
-      (difficulty . 1)
+      (unlocked-by hello-world)
+      (difficulty . 3)
```
Comment (guygastineau) [inline code/config.ss]: I am basically ready to merge this, but I wonder if `two-fer` really warrants difficulty level `3`?

Comment (guygastineau) [reply code/config.ss]: It is quite a bit simpler than `scrabble-score`

Comment (jitwit) [reply code/config.ss]: It's not clear how to do optional arguments in scheme, imo. That was the reason for the elevated difficulty

Comment (jitwit) [reply code/config.ss]: In guile it requires looking up some baroque syntax, in chez it involves looking up `case-lambda`, or understanding how arguments are passed as lists

Comment (jitwit) [reply code/config.ss]: In addition to that, it also involves string concatenation or formatting

Comment (guygastineau) [reply code/config.ss]: Okay, I can live with that 😃

**Pull request merged**